### ISSUE TITLE
[Screen Reader – Make Code - Create a project] Heading 'Create a project' is not defined under<H> tag programmatically.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3560]
**User Experience**:
Users who depend on screen reader will not be able to identify and interact with the element properly, if heading is not defined programmatically for the controls. . 

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing under "My Project" section in Home page and select 'New Project' Button.
4. Navigate all controls under 'Create a Project' pop-up.
5. Verify that heading level is not defined for 'Create a project'. Observe that behavior.

**Actual Result:**
Heading 'Create a project' is not defined under<H> tag programmatically. Screen reader does not announce 'Create a project' as heading.

**Expected Result:**
Heading 'Create a project' should be defined programmatically under<H> tag and Screen reader should announce 'Create a project' as heading .

**Note:**
1. Same issue appear under Chrome & Firefox.
2. Same issue exists throughout the pop-up window in this application.

**WCAG Reference:**
[WCAG 2.4.6 - Headings and Labels
](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.6, #Win10-Edge(Chromium)

![WCAG2 4 6_Create a project](https://user-images.githubusercontent.com/72871089/97958006-574e0280-1dd2-11eb-981f-8acfc3aa9813.jpg)

**Comment by jwunderl** [comment]
duplicate https://github.com/microsoft/pxt-microbit/issues/3560
